src/literal.cc

```diff
@@ -384,6 +384,13 @@ Result parse_float(LiteralType literal_type,
                    const char* s,
                    const char* end,
                    uint32_t* out_bits) {
+#if COMPILER_IS_MSVC
+  if (literal_type == LiteralType::Int && string_starts_with(s, end, "0x"))
+  {
```

Comment:
The style for this repo is to not have opening braces on a line by themselves.

(tip: You can use clang-format to format in accordance with the local style since there is a .clang-format file at the top level of the repo)